easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb

```
@@ -0,0 +1,417 @@
+name = 'Python'
+version = '2.7.18'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['da3080e3b488f648a3d7a4560ddee895284c3380b11d6de75edb986526b9a814']
+
+dependencies = [
+    ('binutils', '2.35'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.33.0'),
+    ('GMP', '6.2.0'),  # required for pycrypto
+    ('libffi', '3.3'),  # required for cryptography
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.1h'),
+]
+
+osdependencies = [OS_PKG_OPENSSL_DEV]
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+# order is important!
+# package versions updated 13 October 2020
+exts_list = [
+    ('pip', '20.2.3', {
+        'use_pip': False,
+        'checksums': ['30c70b6179711a7c4cf76da89e8a0f5282279dfb0278bec7b94134be92543b6d'],
+    }),
+    ('setuptools', '44.1.1', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'use_pip': False,
+        'checksums': ['c67aa55db532a0dadc4d2e20ba9961cbd3ccc84d544e9029699822542b5a476b'],
+    }),
```

Installing `pip` fails because `setuptools` is missing. These two extensions should be reversed as it's done in `Python-2.7.18-GCCcore-9.3.0.eb`.

```suggestion
    # setuptools 44.1.1 is the most recent version still supporting Python 2.x
    ('setuptools', '44.1.1', {
        'source_tmpl': '%(name)s-%(version)s.zip',
        'use_pip': False,
        'checksums': ['c67aa55db532a0dadc4d2e20ba9961cbd3ccc84d544e9029699822542b5a476b'],
    }),
    ('pip', '20.2.3', {
        'use_pip': False,
        'checksums': ['30c70b6179711a7c4cf76da89e8a0f5282279dfb0278bec7b94134be92543b6d'],
    }),
```